src/main/java/io/vertx/core/shareddata/Shareable.java

```diff
@@ -26,4 +26,11 @@
  * @author <a href="http://tfox.org">Tim Fox</a>
  */
 public interface Shareable {
+
+  /**
+   * Returns a copy of the object.
```

Comment:
Can you update the comment to explain that only mutable objects need to provide an implementation of this method?